### ISSUE TITLE
Backup not working

### ISSUE THREAD

**Issue by ElineMaaikedeWeerd** [maas/maas-ansible-playbook#116]
Backup isn't working since the HA postgres branch has landed: postgres backup tasks are missing, and `_primary` names no longer exist.

**Comment by simonlock** [comment]
Yes, I have the very same issue. Renaming maas_postgres_primary to maas_postgres allows the backup.yaml task to progress further but then no archives are created.

**Comment by SK1Y101** [comment]
Additional note: I can't see anything that suggests we backup MAAS snaps? We have the ability to restore them it would seem
